modules/social_features/social_user_export/src/Plugin/UserExportPlugin/UserAddressAdministrative.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Drupal\social_user_export\Plugin\UserExportPlugin;
+
+use Drupal\social_user_export\Plugin\UserExportPluginBase;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\user\UserInterface;
+
+/**
+ * Provides a 'UserAddressAdministrative' user export row.
+ *
+ * @UserExportPlugin(
+ *  id = "user_address_administrative",
+ *  label = @Translation("Administrative address"),
+ *  weight = -380,
+ * )
+ */
+class UserAddressAdministrative extends UserExportPluginBase {
+
+  use StringTranslationTrait;
+
+  /**
+   * Returns the header.
+   *
+   * @return \Drupal\Core\StringTranslation\TranslatableMarkup|string
+   *   The header.
+   */
+  public function getHeader() {
+    return $this->t('Administrative address');
+  }
+
+  /**
+   * Returns the value.
+   *
+   * @param \Drupal\user\UserInterface $entity
+   *   The User entity to get the value from.
+   *
+   * @return string
+   *   The value.
+   */
+  public function getValue(UserInterface $entity) {
+    $profile = $this->getProfile($entity);
+    return $this->profileGetAddressFieldValue('field_profile_address', 'administrative_area', $profile);
```

Comment:
Perhaps we can skip defining the `$profile` variable and just make it:
`return $this->profileGetAddressFieldValue('field_profile_address', 'administrative_area', $this->getProfile($entity));`

Same for other plugins.

Reply:
Allright, good idea.